<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>senales cerebrales</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="campo_grafico" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cargar">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar Señal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_atras">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_adelante">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_aumentar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_disminuir">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desplazamiento en eje X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Escalamiento en Amplitud</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_canal">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar Canal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_filtro">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>460</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campo_canal">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[0-7]</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>390</y>
      <width>81</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lambda</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="boton_minimax">
       <property name="text">
        <string>Minimax</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="boton_univ">
       <property name="text">
        <string>Universal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="boton_sure">
       <property name="text">
        <string>Sure</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>390</y>
      <width>81</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Umbral</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="boton_suave">
       <property name="text">
        <string>Suave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="boton_duro">
       <property name="text">
        <string>Duro</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>390</y>
      <width>91</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ponderación</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="boton_single">
       <property name="text">
        <string>Single</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="boton_one">
       <property name="text">
        <string>One</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="boton_mult">
       <property name="text">
        <string>Multiple</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccione Señal</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="boton_ok">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>110</y>
      <width>270</width>
      <height>169</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="parametrosMetodoDeWelchLabel">
       <property name="text">
        <string>Parámetros Método de Welch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tipoDeVentanaLabel">
       <property name="text">
        <string>Tipo de ventana</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="tipoDeVentanaComboBox1">
       <item>
        <property name="text">
         <string>hanning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hamming</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rectangular</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blackman</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bartlett</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>boxcar</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tamaOLabel">
       <property name="text">
        <string>Tamaño ventana [s]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="welch_tamano">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="solapamientoLabel">
       <property name="text">
        <string>Solapamiento [s]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="welch_solapa">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="frecuenciaInteresLabel">
       <property name="text">
        <string>Frecuencia de muestreo [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fsw">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="graficarw">
       <property name="text">
        <string>Graficar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>120</y>
      <width>261</width>
      <height>263</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="frecuenciaDeInteresLabel">
       <property name="text">
        <string>Frecuencia de muestreo[Hz]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fsm">
       <property name="text">
        <string>250</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tamaOLabel_2">
       <property name="text">
        <string>Ancho de banda[s]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="multiw">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="solapamientoLabel_2">
       <property name="text">
        <string>Efecto banda cercana [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="multit">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="pLabel">
       <property name="text">
        <string>Numero de tapers</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="multip">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="frecuenciaBajaDePasoLabel">
       <property name="text">
        <string>Frecuencia baja de paso [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="flow">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="frecuenciaAltaDePasoLabel">
       <property name="text">
        <string>Frecuencia alta de paso[Hz]</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="fhigh">
       <property name="text">
        <string>40</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="tiempoATomarLabel">
       <property name="text">
        <string>Tiempo a tomar [s]</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="tiempom">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="graficarm">
       <property name="text">
        <string>Graficar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="parametrosMetodoDeWelchLabel_2">
       <property name="text">
        <string>Parámetros Método Multitaper</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>300</y>
      <width>264</width>
      <height>198</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Analisis Tiempo-Frecuencia</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="frecuenciaBajaLabel">
       <property name="text">
        <string>Frecuencia baja [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ft_fl">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="frecuenciaAltaLabel">
       <property name="text">
        <string>Frecuencia alta [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ft_fh">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Tiempoiniciallabel">
       <property name="text">
        <string>Tiempo inicial [s]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ft_ti">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="timepoFinalLabel">
       <property name="text">
        <string>Tiempo final [s]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="ft_tf">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="graficarescalog">
       <property name="text">
        <string>Graficar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="frecuenciaDeMuestreoHzLabel">
       <property name="text">
        <string>Frecuencia de muestreo[Hz]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ft_fs">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1385</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
